Makefile

```diff
@@ -59,14 +59,11 @@ kotsadm:
 # Debugging
 .PHONY: kotsadm-debug-build
 kotsadm-debug-build:
-	go build ${LDFLAGS} -gcflags="all=-N -l" \
- 		$(BUILDFLAGS) \
-		-o ./bin/kotsadm-api-debug \
-		./cmd/kotsadm
+	CGO_ENABLED=1 go build -v -o ./bin/kotsadm-api-debug ./cmd/kotsadm
```

Comment:
Seems odd that we're enabling CGO here and dropping the gcflags

Reply:
I'd have to test this, I wouldn't think that CGO would impact when/if a module is downloaded

Reply:
@syntastical this one is easily testable
run `CGO_ENABLED=0 go build -v -o ./bin/kotsadm-api-debug ./cmd/kotsadm`
and you can see the modules downloading
and again run `CGO_ENABLED=1 go build -v -o ./bin/kotsadm-api-debug ./cmd/kotsadm` 
you can see the modules not downloading
and again run `CGO_ENABLED=0 go build -v -o ./bin/kotsadm-api-debug ./cmd/kotsadm`
you will see the mods downloading again

Reply:
I tested with and without CGO and I didn't notice a need re-download.  I think we should remove this flag.
We should add back the follow ${LDFLAGS} ${GCFLAGS} $(BUILDFLAGS) to the build.  I think this eliminates the need to alter Dockerfile.okteto.  We didn't include these in the skaffold environment so I'm skeptical that they'd be required here.